tests/phpunit/Unit/Graph/GraphFormatterTest.php

```diff
@@ -118,7 +119,7 @@ class GraphFormatterTest extends \PHPUnit\Framework\TestCase {
 	 * @return GraphFormatter
 	 */
 	private static function graph( array $case ): GraphFormatter {
-		$graph = new GraphFormatter( new GraphOptions( GraphFormatterTest::BASE_PARAMS + $case['params'] ) );
+		$graph = new GraphFormatter( new GraphOptions( array_merge( GraphFormatterTest::BASE_PARAMS, $case['params'] ) ) );
```

Comment:
Is there any reason why you changed this?

Reply:
`+` was an error. In this place `$case['params']` should override `GraphFormatterTest::BASE_PARAMS`.